### PR TITLE
Added integration test for frame restarts

### PR DESCRIPTION
**Pull request by kmazurek**

### CODE REVIEW

**Review by shadeofblue — APPROVED** [review]
looks good

File: scripts/node_integration_tests/playbooks/golem/restart_frame.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import typing
+
+from ..base import NodeTestPlaybook
+
+
+class RestartFrame(NodeTestPlaybook):
+    provider_node_script = 'provider/debug'
+    requestor_node_script = 'requestor/debug'
+    requestor_node_script_2 = 'requestor/always_accept_provider'
+    task_settings = 'default'
```
Comment (shadeofblue) [inline scripts/node_integration_tests/playbooks/golem/restart_frame.py:10]: though - we should probably make the task multi-frame and make sure only the specified frame is restarted and all of them...

Comment (kmazurek) [reply scripts/node_integration_tests/playbooks/golem/restart_frame.py:10]: Makes sense, but as discussed - let's make this as a separate change.

**Review by jiivan — APPROVED** [review]
lgtm

**Pull request merged**